src/sass/getters/ranges.scss

```diff
@@ -58,10 +58,11 @@
 	$list: ();
 	@each $range in conf.$percent-range {
 		$list: list.append($list, $range * $end);
+		@if ($begin) {
+			$list: list.append($list, ($range * $end) + $begin);
```

Comment:
I rename `$end` to `$multiply`

If $begin is 1 -> 1, 1.1, 1.2, 1.25, 1.3, 1.4, 1.5, 1.6, 1.7, 1.75, 1.8, 1.9, 2,
if $begin is 0 -> 0, 0.1, 0.2, 0.25, 0.3, 0.4, 0.5, 0.6, 0.7, 0.75, 0.8, 0.9, 1